### DISCUSSION TITLE
Prevent Resources from being imported. But still include them via the Export Configuration.

### DISCUSSION THREAD

**Discussion by Decapitated** [godotengine/godot-proposals#10330] [General Discussions]
I am creating a GDExtension that [renders HTML](https://github.com/Decapitated/Godot-HTML).

The extension loads files via `FileAccess`. When exporting the demo project, I import Non-Resource files via the `Export Configuration` - html, js, css, etc.

This works great.  The files are put inside the `.pck` file, and I am able to access these files via `FileAccess` and a path like `res://folder/file.html`.

The problem arises when files like `SVG` and `PNG` - which are auto imported - are accessed this way. 
According to the docs, `FileAccess` cannot access imported resources.
I would like for resources inside a particular folder to not be imported. But still included inside the `.pck` and accessed like other Non-Resources.
I have attempted to use `.gdignore` inside the folders, which stops the files from being imported. But also prevents the files from being included in the `.pck` via the `Export Configuration`.

**Comment by AThousandShips** [comment]
You just pick "Keep File (exported as is)" in the import option

**Reply by Decapitated** [reply]
You're exactly right. That fixes my problem. Is there anyway to do this without going to each image/resource?

**Reply by AThousandShips** [reply]
Yes you select all of them and do the same you can configure multiple resources at once (as long as they are the same type)

**Reply by Decapitated** [reply]
Thank you! Of course it was a simple solution. 😳
